Plugins/GeoModel/include/Acts/Plugins/GeoModel/GeoModelConverters.hpp

```diff
@@ -68,4 +69,24 @@ using GeoUnionDoubleTrdConverter =
     detail::GenericGeoShapeConverter<GeoShapeUnion,
                                      detail::GeoUnionDoubleTrdConverter>;
 
+/// @brief The map that maps the converters with the shapes
+
+std::shared_ptr<const IGeoShapeConverter> GeoShapesConverters(int geoShapeId) {
+  using ConverterLookUpMap =
+      std::unordered_map<int, std::shared_ptr<const IGeoShapeConverter>>;
```

Comment:
I guess the local typedef is not really needed since you only use it in this location